hail_scripts/reference_data/combine.py

```diff
@@ -156,12 +167,13 @@ def update_existing_joined_hts(
     dataset_ht = get_ht(dataset, genome_version)
     if 'coverage' not in dataset:
         joined_ht = joined_ht.drop(dataset)
+        joined_ht = joined_ht.select_globals(
+            **joined_ht.globals.drop(f'{dataset}_globals'),
+        )
```

Comment:
```suggestion
        joined_ht = joined_ht.drop(f'{dataset}_globals')
```
I may be missing something but you could just do this to drop the global

Reply:
Yes that is WAY better.  🤦